package.json

```diff
@@ -23,10 +23,10 @@
     "@emotion/styled": "^11.6.0",
     "@user-office-software/duo-localisation": "^1.2.0",
     "@user-office-software/duo-validation": "^2.4.3",
-    "@graphql-codegen/cli": "^2.3.0",
-    "@graphql-codegen/typescript": "^2.4.1",
-    "@graphql-codegen/typescript-graphql-request": "^4.3.2",
-    "@graphql-codegen/typescript-operations": "^2.2.1",
+    "@graphql-codegen/cli": "^2.4.0",
```

Comment:
Are you using something from version 2.4.0 or you updated just because it outdated

Reply:
@jekabs-karklins Yes I am glad that they resolved the issue with the maybe values containing undefined. Now the part `maybeValue: T | null` is not needed anymore and I remove it with this PR after upgrading the cli